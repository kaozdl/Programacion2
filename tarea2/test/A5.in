insertar_lista (1,gallo) (2,perro) (3,perro) .
filtrado_lista 2 <
filtrado_lista 2 =
filtrado_lista 2 >
insertar_lista (0,dragon) (1,tigre) .
filtrado_lista 16 =
filtrado_lista 0 =
liberar_lista
Fin
