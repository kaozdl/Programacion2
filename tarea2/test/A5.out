1>(0,primero)(1,gallo)(2,perro)(3,perro)
2>1
(0,primero)(1,gallo)
3>1
(2,perro)
4>1
(3,perro)
5>(0,primero)(0,primero)(1,gallo)(2,perro)(3,perro)(0,dragon)(1,tigre)
6>
7>3
(0,primero)(0,primero)(0,dragon)
8>9>Fin.
